dev/test-studio/schema/standard/portableText/customPlugins.tsx

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable react/jsx-no-bind */
+import {defineBehavior} from '@portabletext/editor/behaviors'
+import {BehaviorPlugin, DecoratorShortcutPlugin, OneLinePlugin} from '@portabletext/editor/plugins'
+import {defineType, type PortableTextInputProps} from 'sanity'
+
+export const customPlugins = defineType({
```

Comment:
Because of the `autoCloseBrackets` Studio Plugin, all PTE Inputs have the `autoCloseBracketsBehavior` by default. However, unfortunately, these local usages of `renderPlugins` take precedence.